lib/model/content.dart

```diff
@@ -986,18 +986,18 @@ class _ZulipContentParser {
           && divElement.className == "codehilite");
 
       if (divElement.nodes.length != 1) return null;
-      final child = divElement.nodes[0];
+      final child = divElement.nodes.first;
       if (child is! dom.Element) return null;
       if (child.localName != 'pre') return null;
 
-      if (child.nodes.length > 2) return null;
+      if (child.nodes.length > 2 || child.nodes.isEmpty) return null;
```

Comment:
nit: include the key "why" information in the commit message, rather than rely on the PR description. We aim for the Git history to be able to serve as our primary record of what we did.

In particular key points include
* this was a crash bug;
* the affected messages in public chat.zulip.org history ranged from message ID 5020 to 29855, all in 2016.

Details that can be read off from the commit's added test case don't need to be repeated in the commit message, though.